### ISSUE TITLE
There is a bug if you give the attribute off hand.

### ISSUE THREAD

**Issue by mortal-immortal** [nvb-uy/CIA#7]
If you give an attribute to the off hand, there is a bug where the off hand attribute is not applied when using an item with the same attribute in the main hand.
Prepare two items with different names and the same ability.
One on the left and one on the right.
Afterwards, empty your main hand.
The off hand ability disappears.

**Comment by ChoxRaven** [comment]
While testing I found the exact same issue, here are the steps I took to reproduce it:

- Get 2 instances of the same item, a stat increase while held in the offhand.
- Have one in your offhand, and the other in your mainhand. This causes the bonus to not work.
- Switch to a different item, or nothing in your mainhand, the bonus still doesn't work.

There was also another issue where items that had modifiers while in your offhand would not show "When in Offhand:" before the offhand stat modifiers. 

my best guess for the former issue is that there is a issue with how the mod handles multiple types of the same item.
